Verso integration blog

Legend-Master opened the pull request:
@dklassic @wusyong

Legend-Master committed:
Initial draft

Legend-Master committed:
Re-recorded the video

Legend-Master committed:
Fix video path

dklassic commented:
@Legend-Master , I believe it would be beneficial to expand on several key points to better inform our readers about our objectives. Specifically, we should elaborate on the following:

- **What Servo Is:** Explain what Servo is and why using the Servo webview is advantageous.
- **Goals of `tauri-runtime-verso`:** Detail what `tauri-runtime-verso` aims to achieve.
- **Future Work:** Outline the future developments planned for this project.

I can assist with developing the first two points and enhancing the existing content. However, you'll need to handle the future work section, as I'm not very familiar with that aspect of the project.

I suggest focusing on:

- **Our Ultimate Goals:** Clearly define what we aim to achieve.
- **Steps to Achieve These Goals:** Outline the necessary actions and milestones.

Once you've updated the article to include the future work section, I can continue refining it to improve the overall reading experience.

Legend-Master committed:
Add future work section

Legend-Master committed:
Typos

Legend-Master commented:
@dklassic I have added the future works section, please take a look and let me know what's missing, also thank you for the help!

dklassic commented:
> @dklassic I have added the future works section, please take a look and let me know what's missing, also thank you for the help!

lgtm! I'll write up an improvement reference from here.

dklassic commented:
@Legend-Master I’ve written a reference that might help you enhance the blog post:

https://github.com/dklassic/tauri-docs/blob/d6331fb444eb1493cb03b4b0f103275da2a7c8da/src/content/docs/blog/tauri-verso-integration.md

Personally, while it's great to include detailed future work, we should be cautious not to overpromise. Overhyping features that might take a long time to develop—or may not materialize at all—can lead to unrealistic expectations within the community. It might be best to tone down some of the future projections to maintain trust and manage expectations effectively.

Legend-Master commented:
Thanks, I'll take a look and apply the changes as I see fit

> Personally, while it's great to include detailed future work, we should be cautious not to overpromise.

Yeah, I'll try limit thing to what we can/will do in the next few months

Legend-Master committed:
Cherry picking from @dklassic 's reference

Co-authored-by: dklassic <dklassic@users.noreply.github.com>

Legend-Master commented:
@dklassic I have cherry picked some changes from your branch, I have removed a few things because of correctness and removed `Goals of tauri-runtime-verso` and `Why Servo and Verso?` because I think we have already said that in the introduction and the `tauri-runtime-verso` section

Please take a look and see if it look good to you 🙏 (nice write up by the way)

dklassic commented:
> @dklassic I have cherry picked some changes from your branch, I have removed a few things because of correctness and removed `Goals of tauri-runtime-verso` and `Why Servo and Verso?` because I think we have already said that in the introduction and the `tauri-runtime-verso` section
> 
> Please take a look and see if it look good to you 🙏 (nice write up by the way)

lgtm! Let's maybe see if @wusyong have any thought about it as well.

Legend-Master committed:
Merge remote-tracking branch 'origin/v2' into verso-integration-blog

vasfvitor commented:
@Legend-Master I merged it but now I'm unsure if there was a timeline to release, or is it ok?

Legend-Master commented:
> @Legend-Master I merged it but now I'm unsure if there was a timeline to release, or is it ok?

Sorry I have to revert this, there're some disagreements that I can't merge this yet, it should be a draft but marked as ready somehow

Legend-Master commented:
@dklassic I have replaced the blog with the original version, and decided not to go with your version, sorry but I see there're too many marketing terms that I just can't get over myself to take it

dklassic commented:
> @dklassic I have replaced the blog with the original version, and decided not to go with your version, sorry but I see there're too many marketing terms that I just can't get over myself to take it

No worries, do whatever your heart desires!
